.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,234 @@
+name: prebuild
+
+on:
+  push:
+    branches:
+      - main
+      - dev-1.x
+      - master
+    paths:
+      - "mmdeploy/version.py"
+
+permissions: write-all
+
+jobs:
+  linux_build:
+    runs-on: [self-hosted, linux-3090]
+    container:
+      image: openmmlab/mmdeploy:manylinux2014_x86_64-cuda11.3
+      options: "--gpus=all --ipc=host"
+      volumes:
+        - /data2/actions-runner/prebuild:/__w/mmdeploy/prebuild
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Get mmdeploy version
+        run: |
+          export MMDEPLOY_VERSION=$(python3 -c "import sys; sys.path.append('mmdeploy');from version import __version__;print(__version__)")
+          echo $MMDEPLOY_VERSION
+          echo "MMDEPLOY_VERSION=$MMDEPLOY_VERSION"  >> $GITHUB_ENV
+      - name: Build MMDeploy
+        run: |
+          source activate mmdeploy-3.6
+          pip install pyyaml packaging setuptools wheel
```

Comment:
In the image "openmmlab/mmdeploy:manylinux2014_x86_64-cuda11.3", `pyyaml`, `setuptools` and `wheel` are already installed.